main.go

```diff
@@ -2,13 +2,18 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"math"
 	"math/rand"
 	"net/url"
 	"os"
 	"time"
 
+	"github.com/elastic/hey-apm/out"
```

Comment:
could clean up these imports

Reply:
seems i keep having import differences, i run `goimports - l -w ./` in a pre-commit hook and that is the result; how do you do it?

Reply:
I use `goimports -local github.com/elastic -w` but also sometimes need to manually group things.